aten/src/ATen/native/native_functions.yaml

```diff
@@ -495,6 +495,30 @@
 - func: randn_like(Tensor self, *, Type dtype) -> Tensor
   variants: function
 
+- func: randint(Type dtype, int64_t high, IntList size, *, Generator* generator=nullptr) -> Tensor
```

Comment:
nit: can you move this above `randn*` so that this file is alphabetically ordered.